Addresses #5984 Doc: Making the negation operator clearer (#5985)
aiidateam/aiida-core: docs/source/howto/query.rst

        },
    )

> .. _how-to:query:filters-operator-negations:

Reviewer: ```suggestion
.. _how-to:query:filters:operator-negations:
```

        },
    )

> You can **negate** a filter by adding an exclamation mark, ``!`` or a ``~``, in front of the operator, that is a ``NOT`` operatror.

Reviewer: Think the original phrasing could be mistaken to mean that the filter itself is no longer considered. Maybe the following is more correct?
```suggestion
A filter can be turned into its associated **negation** by adding an exclamation mark, ``!``, in front of the operator.
```
I also wonder whether we should really be providing two different values even if both work. Are both identical? If so, I would just provide one, to keep things simple. If they behave differently, we should document here how.
